ansible/group_vars/all

```diff
@@ -96,8 +96,10 @@ jmx:
   jvmCommonArgs: "-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.authenticate=true -Dcom.sun.management.jmxremote.password.file=/root/jmxremote.password -Dcom.sun.management.jmxremote.access.file=/root/jmxremote.access"
   enabled: "{{ jmxremote_enabled | default('true') }}"
 
-transactions:
-  stride: "{{ groups['controllers'] | length }}"
+transaction:
+  header: "{{ transaction_header | default('X-Request-ID') }}"
+  # The tid can be any String between 1 and 64 characters.
+  regex: "{{ whisk_tid_regex | default('[a-zA-Z0-9_-]{1,64}') }}"
 
```

Comment:
excluding - would make it easier to copy/paste a code (the dash breaks double click to highlight a string).

Reply:
That's a good idea. This makes operations much easier :)